src/main/webapp/script.js

```diff
@@ -245,11 +269,13 @@ function toggleStartTripInputStage() {
     const startTripSubmitButton = document.getElementById('submit-calculate-trip');
     startTripSubmitButton.style.display = 'inline-block';
 
-    // Change location and date inputs to be readonly.
-    const destinationInput = document.getElementById('inputDestination');
-    destinationInput.readOnly = true;
-    const dayOfTravelInput = document.getElementById('inputDayOfTravel');
-    dayOfTravelInput.readOnly = true;
+    // Change name, location, and date inputs to be readonly.
+    const inputTripName = document.getElementById('inputTripName');
+    inputTripName.readOnly = true;
+    const inputDestination = document.getElementById('inputDestination');
+    inputDestination.readOnly = true;
+    const inputDayOfTravel = document.getElementById('inputDayOfTravel');
+    inputDayOfTravel.readOnly = true;
```

Comment:
I would argue that putting all these elements into their own variables is actually starting to decrease your clarity here. Take 'inputTripName':

1. the name of the variable is the same as you're passing into getElementById, so you aren't actually adding to a user's comprehension of this element by adding a name
2. you aren't ever using this variable again, so it doesn't keep the line count down
3. "document.getElementById('inputTripName').readOnly = 'true'" isn't actually difficult to read and understand.

Reply:
This is a good point, I started to realize this as I kept writing these two lines in pairs. Fixed, thanks for the note!